src/EditorFeatures/CSharpTest2/Recommendations/AndKeywordRecommenderTests.cs

```diff
@@ -92,6 +92,19 @@ void M()
         if (e is System.Collections.Generic.List<int> $$");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
```

Comment:
I realized that I didn't include a recursive pattern test in the initial keyword completion PR, so I added one in this PR.